src/domains/citizen-auth-app/01_monitor.tf

```diff
@@ -31,3 +40,47 @@ data "azurerm_monitor_action_group" "email" {
   resource_group_name = var.monitor_resource_group_name
   name                = local.monitor_action_group_email_name
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert_v2" "samlresponse_missing_detection_alert" {
+  enabled                 = true
+  name                    = "[${upper(var.domain)}] Missing required SAMLResponse in assertionConsumerService"
+  resource_group_name     = data.azurerm_resource_group.monitor_rg.name
+  scopes                  = [data.azurerm_application_gateway.app_gateway.id]
+  description             = <<-EOT
+    Detected multiple SAMLResponse missing during assertionConsumerService.
+    IdP is unknown here but you can watch Mixpanel events to identify the IdP
+    that is causing the alert to trigger
+  EOT
+  severity                = 0
+  auto_mitigation_enabled = false
+  location                = data.azurerm_resource_group.monitor_rg.location
+
+  // check once every 5 minutes(evaluation_frequency)
+  // on the last 10 minutes of data(window_duration)
+  evaluation_frequency = "PT5M"
+  window_duration      = "PT10M"
+
+  criteria {
+    query                   = <<-QUERY
+AzureDiagnostics
+| where originalHost_s in ("app-backend.io.italia.it", "api-app.io.pagopa.it")
+| where requestUri_s == "/error.html"
+| where httpMethod_s == "GET"
+| where requestQuery_s contains "Missing%20SAMLResponse%20in%20ACS"
+    QUERY
+    operator                = "GreaterThanOrEqual"
+    time_aggregation_method = "Count"
+    threshold               = 5
```

Comment:
How did we decided the threshold?

Reply:
some providers have very low login rate so I decided that 5 missing SAMLResponses could have been good for triggering the alert. do you believe I should lower/increment the threshold?